app/views/conversations/show.html.erb

```diff
@@ -37,7 +37,15 @@
   <div class="request-container">
     <div class="conversation-request-card">
       <div class="conversation-request-card-status">
-        <p id="request-status"><%= @request.status.capitalize %></p>
+        <% if @request.helper_id.present? %>
```

Comment:
For this bit - to make it more readable - I think we could to check @request.status instead of checking whether a helper is present, since this information is all about the request status and we show an avatar when the status is anything but pending.

Reply:
Makes sense I'll change that